### DISCUSSION TITLE
Collabration feature of --link

### DISCUSSION THREAD

**Discussion by Cormanz** [coder/code-server#3583] [Ideas]
I think it should be a feature to make it so you can config --link to allow specific other github users to join in on your session personally. That's all

**Comment by jsjoeio** [comment]
That would be super cool!

We do have an [open issue](https://github.com/cdr/code-server/issues/33) to explore collaboration (in general) for code-server. That would probably be the first step.

**Reply by Cormanz** [reply]
Yeah, me and my friend are working on a discord bot. Normally we use live-share but it is not a good experience and I've been a huge fan of code-server (it's awesome, have been using it at school and stuff, sometimes I use it locally over native VSCode) and collaboration is just the one piece of the puzzle that looks like it's missing.

Excited to see your progress! Will watch the collabration issue.
